sanity-check/main.py

```diff
@@ -67,14 +67,15 @@ def main():
                 final_metadata,
                 args.submission_song_url
         )
+
         if not args.force:
             check_analysis_exists(
                 final_metadata,
                 args.submission_song_url
             )
         
         update_tsv(final_metadata,"updated_"+args.experiment_info_tsv)
-        
+    
 
 def load_tsv(experiment_info_tsv):
     metadata_dict = {}
```

Comment:
Should we also check early if any of the following `required` fields are missing or provided with empty string similar as IDs?
- submitter_sequencing_experiment_id
- program_id
- read_group_count
- platform
- experimental_strategy

Reply:
Those checks are in `pGenExp`. We could move them here but that'd would make the later ones redundant.

Reply:
Yes, that makes sense. Let's keep the checks in `pGenExp`. 
No further changes are needed for now.